nio/client/async_client.py

```diff
@@ -559,6 +563,17 @@ async def _handle_joined_rooms(self, response: SyncType) -> None:
         if self.store:
             self.store.save_encrypted_rooms(encrypted_rooms)
 
+    async def _handle_presence_events(self, response: SyncType):
```

Comment:
Besides what @mirukana said, don't we want to call some type of callback for the events here? If no can't we use the `_handle_presence_events` method from the base client?

Reply:
Yeah. Forgot about that. Added callbacks now